app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -159,6 +159,7 @@ def pre_prov
 
   def render_updated_templates
     report_scopes = [:eligible_for_provisioning]
+    report_scopes.push([:filter_with_name, params[:search_text]])
```

Comment:
`:filter_with_name` is associated with the change in here - https://github.com/ManageIQ/manageiq/blob/71899d696b71d8897f38246aa65d82923e65aaae/app/models/miq_template.rb#L35